src/main/java/com/nike/cerberus/service/AuthenticationService.java

```diff
@@ -402,6 +402,27 @@ private VaultAuthResponse generateToken(final String username, final Set<String>
     }
 
     /**
+     * Build a set of policies that is allowed for the specific IAM principal (e.g. an assumed user, or instance profile)
```

Comment:
@fieldju ok, does adding the last line you put clear it up? or should I rephrase the rest as well